src/utils/constants.ts

```diff
@@ -0,0 +1,11 @@
+export const DEFAULT_DEPLOYER = "default";
+export const DEFAULT_RPC_URL = "http://127.0.0.1:8545";
+export const DEFAULT_CHAIN_ID = 31337;
```

Comment:
ok. it's defined here as a constant. maybe use this instead of the magic number?

Reply:
ahh good catch, will update - thanks!

Reply:
should be updated now!